administrator/components/com_admin/script.php

```diff
@@ -895,6 +895,101 @@ public function postflight($action, $installer)
         }
 
         // Add here code which shall be executed only when updating from an older version than 5.0.0
+        if (!$this->migrateTinymceConfiguration()) {
+            return false;
+        }
+
+        return true;
+    }
+
+    /**
+     * Migrate TinyMCE editor plugin configuration
+     *
+     * @return  boolean  True on success
+     *
+     * @since   __DEPLOY_VERSION__
+     */
+    private function migrateTinymceConfiguration(): bool
+    {
+        $db = Factory::getDbo();
+
+        try {
+            // Get the TinyMCE editor plugin's parameters
+            $params = $db->setQuery(
+                $db->getQuery(true)
+                    ->select($db->quoteName('params'))
+                    ->from($db->quoteName('#__extensions'))
+                    ->where($db->quoteName('type') . ' = ' . $db->quote('plugin'))
+                    ->where($db->quoteName('folder') . ' = ' . $db->quote('editors'))
+                    ->where($db->quoteName('element') . ' = ' . $db->quote('tinymce'))
+            )->loadResult();
+        } catch (Exception $e) {
+            echo Text::sprintf('JLIB_DATABASE_ERROR_FUNCTION_FAILED', $e->getCode(), $e->getMessage()) . '<br>';
+
+            return false;
+        }
+
+        $params = json_decode($params, true);
+
+        // If there are no toolbars there is nothing to migrate
+        if (!isset($params['configuration']['toolbars'])) {
+            return true;
+        }
+
+        // Each set has its own toolbar configuration
+        foreach ($params['configuration']['toolbars'] as $setIdx => $toolbarConfig) {
+            // Migrate menu items if there is a menu
+            if (isset($toolbarConfig['menu'])) {
+                /**
+                 * Replace array values with menu item names ("old name" -> "new name"):
+                 * "fontformats"  -> "fontfamily"
+                 * "fontsizes"    -> "fontsize"
+                 * "blockformats" -> "blocks"
+                 * "formats"      -> "styles"
+                 */
+                $params['configuration']['toolbars'][$setIdx]['menu'] = str_replace(
+                    ['fontformats', 'fontsizes', 'blockformats', 'formats'],
+                    ['fontfamily', 'fontsize', 'blocks', 'styles'],
```

Comment:
```suggestion
                    ['fontfamily', 'fontsize', 'blocks', 'styles', 'jtemplate'],
```

Reply:
@dgrammatiko Again asking: Are you sure this applies also here to the menu and not only to the toolbar buttons handled in the next loop?